frontend/client/components/Profile/ProfileContribution.tsx

```diff
@@ -46,7 +46,7 @@ class ProfileContribution extends React.Component<Props> {
       );
     } else if (isExpired) {
       tag = <Tag color="red">Expired</Tag>;
-      // TODO: Link to support
+      // TODO: Link to support (issue #337)
```

Comment:
Whoops, I fixed in #350, so I'll remove this in that PR. Might cause a conflict.